dimod/binary/binary_quadratic_model.py

```diff
@@ -1225,7 +1225,8 @@ def resize(self, n: int):
 
     def scale(self, scalar, ignored_variables=None, ignored_interactions=None,
               ignore_offset=False):
-        """Multiply all the biases by the specified scalar.
+        """Multiply all the biases by the specified scalar. Default to C++
```

Comment:
I would not include the default/fallback behavior in the first line of the docstring, see [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).

I probably wouldn't document this implementation detail at all, though if you want to include it, the second paragraph would be the appropriate place.

Reply:
Will remove